### ISSUE TITLE
Have a way to format the metrics in TQDMProgressBar

### ISSUE THREAD

**Issue by zaccharieramzi** [tensorflow/addons#708]
**Describe the feature and the current behavior/state.**
Right now it's not possible to have the metrics displayed in the format you want using `TQDMProgressBar`. I haven't tested it because of #707 , but I know that once that is passed this will be a problem for me. I need my metrics in scientific formatting otherwise they are very difficult to read.

Right now the formatting is fixed: https://github.com/tensorflow/addons/blob/master/tensorflow_addons/callbacks/tqdm_progress_bar.py#L199

**Relevant information**
- Are you willing to contribute it: yes
- Are you willing to maintain it going forward?: no
- Is there a relevant academic paper?: no
- Is there already an implementation in another framework?: yes, in `keras-tqdm`, specifically [here](https://github.com/bstriner/keras-tqdm/blob/master/keras_tqdm/tqdm_callback.py#L13) and [here](https://github.com/bstriner/keras-tqdm/blob/master/keras_tqdm/tqdm_callback.py#L148-L153).
- Was it part of tf.contrib?: I don't think so

**Which API type would this fall under (layer, metric, optimizer, etc.)**
callbacks

**Who will benefit with this feature?**
Everyone using metrics that don't have a nice display when formatted with `.4f`.

**Comment by shun-lin** [comment]
Hi @zaccharieramzi,

Thank you so much, looking forward for your contribution! Let me know if you have any questions.

**Comment by lminer** [comment]
This would be a great feature to have.

**Comment by lminer** [comment]
Just made a PR for this: https://github.com/tensorflow/addons/pull/916
